DuckDuckGo/HomeViewController.swift

```diff
@@ -107,17 +107,15 @@ class HomeViewController: UIViewController {
         delegate?.showSettings(self)
     }
     
-    override func viewWillAppear(_ animated: Bool) {
-        super.viewWillAppear(animated)
-
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+        
```

Comment:
changed this from will to did as it makes the animation a bit cleaner and also a better point to hook in the home screen tips